### DISCUSSION TITLE
Very low performance (on arm64?)

### DISCUSSION THREAD

**Discussion by gthb96** [nextcloud/all-in-one#5172] [Questions] [answered]
Yesterday I deployed most basic AiO on netcup arm64 6 Core 8GB machine.
Suddenly up and download (browser) was insanely slow (50-60mbit)
The machine was basically unloaded and I saw no bottleneck anywhere.

I then deployed Seafile and the speeds now exceed 1GBit up and down.

What could have gone wrong there?
Is it an arm64 issue?

**Comment by szaimen** [comment, marked as answer]
Hi, 

> Suddenly up and download (browser) was insanely slow (50-60mbit)

Does this mean it was faster before it suddenly went slow? What did you do in between with the installation or server?

Can you follow https://github.com/nextcloud/all-in-one/discussions/1335?
